### PR TITLE
Javascript data model

### PR DESCRIPTION
**Pull request by Henry-Nichols**
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+var order = {
+  items: [item],
+  orderId: String,
+  orderDate: Date,
+  orderTotal: Number,
+  shipTo: String,
+  deliveryDate: Date,
+  deliveryNotes: String
+};
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:17]: These can be removed. Data types do not need to be declared in advance in JavaScript

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Very nice work, dude! One minor thing to fix:

File: javascript-data-model/order-history.js (hunk 1 above)
Comment (Henry-Nichols) [reply javascript-data-model/order-history.js:17]: Really? Interesting okay thank you for the feedback Cody! I will fix that right now.

**Review by codingcodymiller — APPROVED** [review]
Great work, man!!

**Pull request merged**